sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/Docs/QuestionAnsweringAuthoringClient.xml

```diff
@@ -1171,7 +1171,7 @@ Schema for <c>QnaSourcesMetadata</c>:
 <code>{
   displayName: string, # Optional. Friendly name of the Source.
   source: string, # Optional. Unique source identifier. Name of the file if it&apos;s a &apos;file&apos; source; otherwise, the complete URL if it&apos;s a &apos;url&apos; source.
-  sourceUri: string, # Required. URI location for the file or url.
+  sourceUri: Uri, # Required. URI location for the file or url.
```

Comment:
This is a breaking change since many of these already GA'd. /cc @KrzysztofCwalina

Reply:
We don't have any API changes. This only impacts document. (we don't have document customization feature).

What's more, the model generated by `string` property will not break when we serialize it even we change it to `Uri`.

Reply:
This is a breaking change if we have model classes, though. You can't change the property type. Does this only affect modeless DPG? /cc @m-nash

Reply:
The pattern of document only happens in DPG. And if it generates models in the future, I think we could do the customization?

Reply:
If something is supposed to be a URI, when we do model classes in the future it should remain a `Uri`. Is that the plan? If so, I can approve for files I owned but you'd should get reviews from a couple others at least - especially for an owner or two of impacted GA libraries.

Reply:
> Is that the plan?

Yes.

@m-nash @lmazuel could you review this change after we upgrade the version to 4.25.0

Reply:
@heaths there was a bug in M4 that would incorrectly keep some properties as `string` while they were having `format: uri`. The change on itself is good and expected, and I would suggest that if some public types are indeed impacted, they are solved by transforms back to a string, as M4 is doing here what it should have done by default since the beginning, so I want new generation to have the right type by default.